walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/datasource/network/apis/HttpAgentPresentationApis.kt

```diff
@@ -0,0 +1,55 @@
+package com.microsoft.walletlibrary.did.sdk.datasource.network.apis
+
+import com.microsoft.walletlibrary.did.sdk.util.Constants
+import com.microsoft.walletlibrary.did.sdk.util.HttpAgentUtils
+import com.microsoft.walletlibrary.util.http.URLFormEncoding
+import com.microsoft.walletlibrary.util.http.httpagent.IHttpAgent
+import com.microsoft.walletlibrary.util.http.httpagent.IResponse
+
+internal class HttpAgentPresentationApis(private val agent: IHttpAgent, private val httpAgentUtils: HttpAgentUtils) {
+
```

Comment:
Isn't a method needed to convert generic response to Presentation specific ones?